tests/integration/ipsec_test.py

```diff
@@ -590,6 +590,38 @@ def test_ipsec_ipv4_libreswan_cert_auth_add_and_remove(
     )
 
 
+def test_ipsec_ipv4_libreswan_remote_cert_auth_add_and_remove(
+    ipsec_hosta_conn_cleanup,
+):
+    desired_state = yaml.load(
+        f"""---
+        interfaces:
+        - name: hosta_conn
+          type: ipsec
+          ipv4:
+            enabled: true
+            dhcp: true
+          libreswan:
+            left: {HOSTA_IPV4_CRT}
+            leftid: '%fromcert'
+            leftcert: hosta.example.org
+            right: {HOSTB_IPV4_CRT}
+            rightid: '%fromcert'
+            rightcert: hostb.example.org
+            ikev2: insist
+            ikelifetime: 24h
+            salifetime: 24h""",
+        Loader=yaml.SafeLoader,
+    )
+    libnmstate.apply(desired_state)
+    assert retry_till_true_or_timeout(
+        RETRY_COUNT, _check_ipsec, HOSTA_IPV4_CRT, HOSTB_IPV4_CRT
```

Comment:
Is there a way to check the `rightcert` is configured as specified ?

Reply:
The only way I can think of is checking the NetworkManager profile to see if it's present.

Reply:
A valid test is using a cert with IP/hostname mismatch with `rightid` or `right`, but that is overkill. Current test is enough to me.